frame/evm/src/lib.rs

```diff
@@ -131,7 +131,7 @@ pub trait Trait: frame_system::Trait + pallet_timestamp::Trait {
 }
 
 decl_storage! {
-	trait Store for Module<T: Trait> as Example {
+	trait Store for Module<T: Trait> as EVM {
```

Comment:
Ah my bad!

Reply:
no worries, you are not the only one obviously, and we should make this better. :)